app/src/main/java/org/mozilla/focus/telemetry/SentryIntegration.kt

```diff
@@ -50,6 +50,39 @@ object SentryWrapper {
         // they're personally identifiable information.
         Sentry.getContext().user = SentryUnknownUser()
     }
+
+    /**
+     * One way to test calls to static methods is to pass instances of their objects into the
```

Comment:
See my other comment about the first line of kdoc comments.

In general, this comment seems unrelated to what the method does so is inappropriate for kdoc. In situations like these, prefer comments that are not exposed in API documentation, like `//`. (we generally don't use `/*` comments).